apps/explorer/src/app/components/txs/txs-stats-info.tsx

```diff
@@ -0,0 +1,79 @@
+import { gql, useQuery } from '@apollo/client';
+import { t } from '@vegaprotocol/react-helpers';
+import { useEffect } from 'react';
+import { InfoBlock } from '../../components/info-block';
+import type { TxsStats, TxsStats_statistics } from './__generated__/TxsStats';
+import classNames from 'classnames';
+
+const STATS_QUERY = gql`
```

Comment:
chore (blocking): please try to stay away from writing queries inline, we're trying to migrate the type generation to use a new library, as apollo's type generator is marked as deprecated. all you have to do is to put your query in a `.graphql` file, run the type gen, and import the generated typed hook from there.

Reply:
@elmar-vega i don't understand, can you explain what is causing difficulties using the new method here over the old one?

Reply:
alright, up to you, happy to approve it if you say so.